add breaking change information about the return of a resource (#847)
gravitee-io/gravitee-docs: pages/apim/3.x/installation-guide/upgrades/3.18.8/README.adoc

= Upgrade to 3.18.8

== Breaking changes

> === Return type of /apis/{apiId}/subscribers has changed

Reviewer: ```suggestion
=== Change of return type when calling /apis/{apiId}/subscribers
```

Author: fixed

> In order to improve performances while fetching the subscribers of an API, internal methods have changed and the output has evolved.

Reviewer: ```suggestion
To improve performance while fetching the subscribers of an API, internal methods have changed and the output has evolved.
```

> When calling `/organizations/{orgId}/environments/{envId}/apis/{apiId}/subscribers`, you will now get a list of `ApplicationListItem` instead of `ApplicationEntity`.

Reviewer: ```suggestion
As a result, a call to `/organizations/{orgId}/environments/{envId}/apis/{apiId}/subscribers` now returns a list of `ApplicationListItem` instead of `ApplicationEntity`.
```

> The `ApplicationListItem` class contains at least the same fields as the `ApplicationEntity` class.

Reviewer: ```suggestion
The `ApplicationListItem` class contains all the fields present in the `ApplicationEntity` class.
```

Author: fixed
